src/main/templates/net/imagej/ops/Ops.list

```diff
@@ -146,6 +146,9 @@ namespaces = ```
 		[name: "textureHomogeneity",          iface: "TextureHomogeneity"],
 		[name: "variance",                    iface: "Variance"],
 	]],
+	[name: "hog", iface: "HoG", ops: [
```

Comment:
actually it should be part of the `feature` `Namespace`, which doesn't exist yet. It should be `features().hog()`.